src/Symfony/Component/HttpFoundation/Tests/UriSignerTest.php

```diff
@@ -9,11 +9,11 @@
  * file that was distributed with this source code.
  */
 
-namespace Symfony\Component\HttpKernel\Tests;
+namespace Symfony\Component\HttpFoundation\Tests;
 
 use PHPUnit\Framework\TestCase;
 use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\HttpKernel\UriSigner;
+use Symfony\Component\HttpFoundation\UriSigner;
 
 class UriSignerTest extends TestCase
```

Comment:
should be duplicated, not moved
the legacy one should be marked `@group legacy`

Reply:
I see this is now marked as legacy, but it still is moved. The file should exist in both HttpKernel and HttpFoundation, with the one in HttpKernel having `@group legacy`